File: push/src/main/java/com/xiaomi/mipush/sdk/stat/db/DataBaseConfig.java
package com.xiaomi.mipush.sdk.stat.db;

import com.xiaomi.slim.Blob;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/stat/db/DataBaseConfig.class */
public class DataBaseConfig {
    public static String DATABASE_NAME = "MessageInfo.db";
    private static final long DEFAULT_MAX_DB_SIZE = 52428800;
    public static long MAX_DB_SIZE = DEFAULT_MAX_DB_SIZE;
    public static int DEFAULT_NUM = Blob.ERROR_INVALID_CHID;
}

File: legacy/src/main/java/com/xiaomi/mipush/sdk/stat/db/DataBaseConfig.kt
package com.xiaomi.mipush.sdk.stat.db
import io.github.magisk317.mipush.protocol.model.*

object DataBaseConfig {
    var DATABASE_NAME: String = "MessageInfo.db"
    private const val DEFAULT_MAX_DB_SIZE: Long = 52428800
    var MAX_DB_SIZE: Long = DEFAULT_MAX_DB_SIZE
    var DEFAULT_NUM: Int = 200
}
